assets/src/editor/events/dates-and-times/editor-date/action-handlers/with-copy-date-entity.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External imports
+ */
+import { isEmpty } from 'lodash';
+import { withDispatch } from '@wordpress/data';
+import { isModelEntityOfModel } from '@eventespresso/validators';
+import { createHigherOrderComponent } from '@wordpress/compose';
+
+const withCopyDateEntity = createHigherOrderComponent(
+	withDispatch( (
+		dispatch,
+		{
+			eventEntity,
+			dateEntity,
+			dateTicketEntities,
+			dateTicketEntitiesLoaded,
```

Comment:
> you changed eventDate to just Date everywhere because you didn't want the extra qualifier in the name,
but then used dateTicket here instead of just ticket.

This is actually not intended,  it _should_ be `ticketEntities` and `ticketEntitiesLoaded`.  An initial iteration had the qualifiers on all props when I realized the drawbacks to doing that.  I simply missed changing these so I'll do that (it's likely buggy as is also).

Reply:
> I think one place where we should use the extra qualifiers is when the returned value contains data from more than one model entity type.

I think we need to get away from passing around maps like this.  It creates an alternate source of truth for relations which can lead to hard to troubleshoot bugs if the map gets stale compared with what _should_ be the source of truth which is the data store.  So generally, I'm looking for ways to _remove_ the need for maps.